neo-tools/sync-with-dmlc.py

```diff
@@ -0,0 +1,73 @@
+import argparse
+import logging
+
+# Install via: pip install GitPython
+from git import Repo
+
+logging.basicConfig(level=logging.INFO)
+
+def parse_args():
+    parser = argparse.ArgumentParser(description='Pull latest commits from dmlc/tvm')
```

Comment:
how about add --dmlc_commit_id as a parameter? we can store dmlc_tvm_commit_id in a wiki or in an issue

Reply:
We better version control this commit id in the repo since so in the future we can fully automate the workflow. Imagine in the future, a robot just needs to clone the repo and run the script.